src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -741,6 +742,15 @@ public class SimpleEvents {
 				)
 				.since("2.7");
 
+		if(Skript.classExists("org.bukkit.event.command.UnknownCommandEvent")) {
+			Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "[player] (wrong|unknown) (cmd|command) [send|use]")
```

Comment:
can you please change the pattern to `unknown command`? this would match EvtCommand, and in my opinion `wrong command` could mean many different things.